packages/rollup-plugin/package.json

```diff
@@ -63,7 +63,7 @@
     "vitest": "^1.5.0"
   },
   "peerDependencies": {
-    "rollup": "3.x || 4.x"
+    "rollup": ">=3.29.5 <4.0.0 || >=4.22.4 <5.0.0"
```

Comment:
@suejung-sentry are we able to rollback to the prior implementation of peer deps declarations? 

With these changes, we'd have to move from a `patch` to a `major` release as we're requiring users to update their version of `rollup`. If they don't, peer dep resolution would fail while they try to install the plugin.

Reply:
goot catch - fixed!